goals.md

```diff
@@ -9,31 +9,56 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## [PROPOSED] 2022
+# 📌[PROPOSED] 2022
 
-### 1 Cultivate growth and adoption of OSPOs
+2022 Strategic Goals have been divided between TODO Steering Committee and LF Staff goals. Please note that some of these goals may be the same for both. 
+
+## Steering Committee SG:
+
+### ➡️ Cultivate growth and adoption of OSPOs
+
+- Make it easier to produce high quality TODO group guides and OSPO Use Cases
+- Host 8 OSPOlogy sessions
+- Encourage more active participation in OSPOlogy
+
+### ➡️ Grow the community
+
+- Cultivate relationships with other communities
+
+### ➡️ Create value for all members
+
+- Run at least 2 surveys to get feedback, >80% satisfaction
```

Comment:
I would rephrase this to "Run at least 2 TODO Group member feedback surveys ..." This will help people understand that you aren't talking about the other survey.